pkgs/applications/misc/logseq/linux.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchurl
+, appimageTools
+, makeWrapper
+  # graphs will not sync without matching upstream's major electron version
+, electron_24
```

Comment:
Electron version 24.8.6 is EOL

Reply:
I am not very familiar with Electron. As mentioned in the link [release 0.9.19](https://github.com/logseq/logseq/blob/bd36efe4f7f5b56a59a160de75708270591ffc55/package.json#L113,) the Electron version for logseq 0.9.19 is still Electron 24. Therefore, it is necessary to use Electron 24. Alternatively,we can wait for the next release of logseq, where the Electron version in the master branch is already Electron 26.

Reply:
from `nixpkgs` point of view electron_24 is marked as insecure due to it's EOL here are your options ( in no particular order )

 - you can try to see if it works with electron_25.
 -  wait for next release, 
 - cherry-pick enough patches from master to get it working with electron 26,
 - package unreleased commit, by marking version as "unstable" ( see contribution guidelines for more )